app/assets/javascripts/initializers/initializeVideoPlayback.js

```diff
@@ -116,6 +118,9 @@ function initializeVideoPlayback() {
       url: metadata.video_source_url,
       seconds: currentTime,
     });
+
+    var properties = { article: metadata.id, deviceType: deviceType };
+    ahoy.track('Article video play', properties);
```

Comment:
This one makes more sense to me since it looks like it should only fire when [the player element is clicked](https://github.com/forem/forem/blob/2b219d420a97c55220c25ec64e1d7bccc03cf11a/app/assets/javascripts/initializers/initializeVideoPlayback.js#L170)! 👍 

My only suggestion is that we change the event name to match what we're doing elsewhere in the codebase with ahoy, for example [here](https://github.com/forem/forem/blob/8425c847c3003caa2516dec36a62b1d06206aaf9/app/views/notifications/index.html.erb#L38). So, the equivalent version would be something like:
```suggestion
    ahoy.track('Clicked Video Player', properties);
```